src/SmartComponents/Recs/Recs.js

```diff
@@ -22,6 +22,11 @@ const ClassicRedirect = lazy(() =>
 const suspenseHelper = (component) => (
   <Suspense fallback={<Loading />}>{component}</Suspense>
 );
+
+const SuspenseHelper = ({ children }) => (
```

Comment:
I'm not sure if this is what makes it behave normally, but either way we could wrap the remaining route components with this as well and remove the `suspenseHelper` function above.